src/main/scala/package.scala

```diff
@@ -113,7 +113,7 @@ package object constellation extends POWExt with SignHelpExt with KeySerializeJS
   }
 
   implicit class KryoSerExt(anyRef: AnyRef) {
-
+    def kryo(ref: AnyRef) = KryoSerializer.serializeAnyRef(ref)
```

Comment:
Why it's needed? 🤔 We should limit the usage of such implicit dependencies as much as possible